HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -4010,6 +4079,202 @@ def from_oga(ceiling_fan)
     end
   end
 
+  class Pools < BaseArrayElement
+    def add(**kwargs)
+      self << Pool.new(@hpxml_object, **kwargs)
+    end
+
+    def from_oga(hpxml)
+      return if hpxml.nil?
+
+      XMLHelper.get_elements(hpxml, 'Building/BuildingDetails/Pools/Pool').each do |pool|
+        self << Pool.new(@hpxml_object, pool)
+      end
+    end
+  end
+
+  class Pool < BaseElement
+    ATTRS = [:id, :heater_id, :heater_type, :heater_annual_energy, :heater_usage_multiplier,
+             :pump_id, :pump_annual_energy, :pump_usage_multiplier,
+             :weekday_fractions, :weekend_fractions, :monthly_multipliers]
+    attr_accessor(*ATTRS)
+
+    def delete
+      @hpxml_object.pools.delete(self)
+    end
+
+    def check_for_errors
+      errors = []
+      return errors
+    end
+
+    def to_oga(doc)
+      return if nil?
+
+      pools = XMLHelper.create_elements_as_needed(doc, ['HPXML', 'Building', 'BuildingDetails', 'Pools'])
+      pool = XMLHelper.add_element(pools, 'Pool')
+      sys_id = XMLHelper.add_element(pool, 'SystemIdentifier')
+      XMLHelper.add_attribute(sys_id, 'id', @id)
+      if not @pump_annual_energy.nil?
+        pumps = XMLHelper.add_element(pool, 'PoolPumps')
+        pool_pump = XMLHelper.add_element(pumps, 'PoolPump')
+        sys_id = XMLHelper.add_element(pool_pump, 'SystemIdentifier')
+        if not @pump_id.nil?
+          XMLHelper.add_attribute(sys_id, 'id', @pump_id)
+        else
+          XMLHelper.add_attribute(sys_id, 'id', @id + 'Pump')
+        end
+        load = XMLHelper.add_element(pool_pump, 'Load')
+        XMLHelper.add_element(load, 'Units', 'kWh/year')
+        XMLHelper.add_element(load, 'Value', to_float(@pump_annual_energy))
+        HPXML::add_extension(parent: pool_pump,
+                             extensions: { 'UsageMultiplier' => to_float_or_nil(@pump_usage_multiplier) })
+      end
+      if not @heater_type.nil?
+        heater = XMLHelper.add_element(pool, 'Heater')
+        sys_id = XMLHelper.add_element(heater, 'SystemIdentifier')
+        if not @heater_id.nil?
+          XMLHelper.add_attribute(sys_id, 'id', @heater_id)
+        else
+          XMLHelper.add_attribute(sys_id, 'id', @id + 'Heater')
+        end
+        XMLHelper.add_element(heater, 'Type', @heater_type)
+        if not @heater_annual_energy.nil?
+          load = XMLHelper.add_element(heater, 'Load')
+          if @heater_type.include?('elec')
+            XMLHelper.add_element(load, 'Units', 'kWh/year')
+          else
+            XMLHelper.add_element(load, 'Units', 'therm/year')
+          end
+          XMLHelper.add_element(load, 'Value', to_float(@heater_annual_energy))
```

Comment:
Change `heater_annual_energy` to `heater_kwh_per_year` and `heater_therm_per_year`. Same for hot tub.